projects/models.py

```diff
@@ -131,10 +131,14 @@ class Project(ModelBase):
         ],
         default=1
     )
+
+    BILLABLE = 0
+    NON_BILLABLE = 1
+
     billable = models.IntegerField(
         help_text='Whether or not the project is chargeable to a'
         ' non-18F client.',
-        choices=[(0, 'Billable'), (1, 'Non-billable')],
+        choices=[(BILLABLE, 'Billable'), (NON_BILLABLE, 'Non-billable')],
```

Comment:
This is unintuitive to me--is there a reason we didn't make `billable` a boolean field?  And even if it *has* to be a choice field, it's a bit unintuitive that billable is `0` and non-billable is `1`. This means that the following code:

```py
if project.billable:
```

will actually be checking if the project is NOT billable... :confounded: